tests/phpunit/tests/functions/TestConfigWpHome.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Test the _config_wp_home function
+ *
+ * @group Functions
+ * @covers ::_config_wp_home
```

Comment:
```suggestion
 * Tests the _config_wp_home() function.
 *
 * @group functions.php
 *
 * @covers ::_config_wp_home
```
- Docblock descriptions should use third-person singular verbs (i.e. "Tests", not "Test").
- The correct group name is `functions.php`.
- See [this ticket](https://core.trac.wordpress.org/ticket/56070) (the second code block) for the ordering and spacing for annotations in the test suite.